frontend/src/components/Shared/SearchBar/SearchBar.jsx

```
@@ -1,43 +1,466 @@
   };
+  
 
+  const renderSuggestionItem = (suggestion, index) => {
+    const suggestionId = `suggestion-item-${suggestion.replace(/\s+/g, '-')}-${index}`;
+    
+    return (
+      <button
+        key={suggestionId}
+        className="w-full px-4 py-2 hover:bg-gray-700 cursor-pointer flex items-center text-gray-300 hover:text-white transition-colors text-left"
+        onClick={() => handleSuggestionClick(suggestion)}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)